WordPress/Classes/Utility/App Configuration/WidgetConfiguration.swift

```diff
@@ -28,9 +28,22 @@ import Foundation
 
             /// Lock Screen
             @objc static let lockScreenTodayViewsKind = "WordPressLockScreenWidgetTodayViews"
-            @objc static let lockScreenTodayLikesCommentsKind = "WordPressLockScreenWidgetTodayLikesComments"
             @objc static let lockScreenTodayViewsProperties = "WordPressLockScreenWidgetTodayViewsProperties"
+
+            @objc static let lockScreenTodayLikesCommentsKind = "WordPressLockScreenWidgetTodayLikesComments"
             @objc static let lockScreenTodayLikesCommentsProperties = "WordPressLockScreenWidgetTodayLikesCommentsProperties"
+
+            @objc static let lockScreenTodayViewsVisitorsKind = "WordPressLockScreenWidgetTodayViewsVisitors"
+            @objc static let lockScreenTodayViewsVisitorsProperties = "WordPressLockScreenWidgetTodayViewsVisitorsProperties"
+
+            @objc static let lockScreenAllTimeViewsKind = "WordPressLockScreenWidgetAllTimeViews"
+            @objc static let lockScreenAllTimeViewsProperties = "WordPressLockScreenWidgetAllTimeViewsProperties"
+
+            @objc static let lockScreenAllTimeViewsVisitorsKind = "WordPressLockScreenWidgetAllTimeViewsVisitors"
+            @objc static let lockScreenAllTimeViewsVisitorsProperties = "WordPressLockScreenWidgetAllTimeViewsVisitorsProperties"
+
+            @objc static let lockScreenAllTimePostsBestViewsKind = "WordPressLockScreenWidgetAllTimePostsBestViews"
+            @objc static let lockScreenAllTimePostsBestViewsProperties = "WordPressLockScreenWidgetAllTimeBestViewsProperties"
```

Comment:
Agree, I have it in TODO list to remove WordPress app widgets and related code altogether. I think it's time. For now, I had to add them here to avoid compiler errors.

Reply:
Great! Thanks for taking care of it @staskus!